service/src/main/kotlin/fi/espoo/evaka/invoicing/service/NewDraftInvoiceGenerator.kt

```diff
@@ -316,6 +341,16 @@ class NewDraftInvoiceGenerator(private val productProvider: InvoiceProductProvid
                 unitId = codes.unitId,
                 product = product
             )
+        ) + surplusContractDays(
+            period,
+            child,
+            product,
+            price,
+            codes,
+            dailyFeeDivisor,
+            contractDaysPerMonth,
+            attendanceDates,
+            absences
```

Comment:
Are fee alterations applied to the daily price for surplus days?

Reply:
Fixed